edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+## WCA Disciplinary Crash Course
+
+# Disciplinary
+
+The WCA Disciplinary Committee (WDC) is entitled to apply bans and other disciplinary action as a result of violation of the [WCA Code of Conduct](https://www.worldcubeassociation.org/documents/Code%20of%20Conduct.pdf), based on the motion [Suspensions and other Sanctions](https://www.worldcubeassociation.org/documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf). However, the Committee is not able to handle difficult situations quickly during a competition, especially if communication with multiple involved parties is required. Cooperation with competition Delegates is often necessary. This document [or chapter, or section, or whatever this is going to be] contains a collection of guidelines to instruct new Delegates on how to handle common disciplinary incidents, what disciplinary measures they can take and what they should report to the WDC.
```

Comment:
> [or chapter, or section, or whatever this is going to be]

Since it has been determined what this is going to be, this parenthetical should be removed.

> ...on how to handle common disciplinary incidents, what disciplinary measures they can take and what they should report to the WDC.

nit: I like Oxford commas.